components/datatypes/agentic/conversation.schema.json

```diff
@@ -0,0 +1,188 @@
+          "title": "Interaction ID",
+          "type": "string",
+          "description": "A single instance of a prompt, response, and feedback"
+        }
+      }
+    },
+    "count": {
+      "properties": {
+        "xdm:count": {
+          "title": "Count",
+          "type": "integer",
+          "description": "Number of prompts/responses/feedback elements in this event, should be equal to 1",
+          "minimum": 1,
+          "maximum": 1
+        }
+      }
+    },
+    "source": {
+      "properties": {
+        "xdm:source": {
+          "title": "Source",
+          "type": "string",
+          "description": "Source of this element",
+          "examples": ["end-user", "bot", "canned"]
+        }
+      }
+    },
+    "raw": {
+      "properties": {
+        "xdm:raw": {
+          "title": "Raw Text",
+          "type": "string",
+          "description": "Raw text element of a conversation"
+        }
+      }
+    },
+    "operations": {
+      "properties": {
+        "xdm:operations": {
+          "title": "Conversational Operations",
+          "type": "object",
+          "description": "Object containing operational information",
+          "properties": {
+            "xdm:cached": {
+              "title": "Cached",
+              "type": "boolean",
+              "description": "Whether this conversation element was cached"
+            }
+          }
+        }
+      }
+    },
+    "feedback-rating": {
+      "properties": {
+        "xdm:rating": {
+          "title": "Rating",
+          "type": "object",
+          "description": "Rating data for the feedback",
+          "properties": {
+            "xdm:score": {
+              "title": "Rating Score",
+              "type": "number",
+              "description": "Numerical value of the rating provided by the end-user. For example: 0, 1 might be used for thumbs up/down."
+            },
+            "xdm:classification": {
+              "title": "Rating Classification",
+              "type": "string",
+              "description": "String/enum value of the rating provided by the end-user.",
+              "examples": [
+                "Thumbs Up",
+                "Thumbs Down",
+                "Positive",
+                "Negative",
+                "5-stars",
+                "1-star"
+              ]
+            },
+            "xdm:reasons": {
+              "title": "Rating Reasons",
+              "type": "array",
+              "description": "Reasons why the customer provided this rating.",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "prompt": {
+      "properties": {
+        "xdm:prompt": {
+          "title": "Prompt",
+          "type": "object",
+          "description": "Object representing a conversation prompt",
+          "allOf": [
+            { "$ref": "#/definitions/count" },
+            { "$ref": "#/definitions/source" },
```

Comment:
Could we make sure that "prompt" has a `query` property that represents the Raw Text.